packages/lesswrong/lib/collections/sideCommentCaches/collection.ts

```diff
@@ -0,0 +1,21 @@
+import { createCollection } from "../../vulcan-lib";
+import { addUniversalFields, getDefaultResolvers } from "../../collectionUtils"
+import { getDefaultMutations } from "../../vulcan-core/default_mutations";
+import { ensureIndex } from "../../collectionIndexUtils";
+import schema from "./schema";
+
+export const SideCommentCaches: SideCommentCachesCollection = createCollection({
+  collectionName: "SideCommentCaches",
+  typeName: "SideCommentCache",
+  schema,
+  resolvers: getDefaultResolvers("SideCommentCaches"),
+  mutations: getDefaultMutations("SideCommentCaches"),
```

Comment:
Possibly we should add collection-level admin-gating, in addition to denying permissions to all individual fields?